[Jan 08 17:22:06 2021 | info] aiogram.dispatcher.dispatcher: Start polling.
[Jan 08 17:22:19 2021 | error] asyncio: Task exception was never retrieved
future: <Task finished name='Task-19' coro=<Dispatcher._process_polling_updates() done, defined at C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py:380> exception=TypeError('not all arguments converted during string formatting')>
Traceback (most recent call last):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 388, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 225, in process_updates
    return await asyncio.gather(*tasks)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\dispatcher.py", line 246, in process_update
    return await self.message_handlers.notify(update.message)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 615, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 615, in notifications_handler
    keyboard = getFinderKeyboard(expresion, 1, database, dbcursor)
  File "C:\Users\Dashedman\github\cifra_bot\main.py", line 236, in getFinderKeyboard
    cur.execute(f"""
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pymysql\cursors.py", line 161, in execute
    query = self.mogrify(query, args)
  File "C:\Users\Dashedman\AppData\Local\Programs\Python\Python38-32\lib\site-packages\pymysql\cursors.py", line 140, in mogrify
    query = query % self._escape_args(args, conn)
TypeError: not all arguments converted during string formatting
[Jan 08 17:22:53 2021 | info] bot: Shutdown... Please wait!
[Jan 08 17:22:53 2021 | info] aiogram.dispatcher.dispatcher: Stop polling...
[Jan 08 17:22:53 2021 | warning] aiogram: Goodbye!
[Jan 08 17:23:04 2021 | info] bot: Starting...
[Jan 08 17:23:05 2021 | info] aiogram: Bot: Cifra [@cifra_s_bot]
[Jan 08 17:23:07 2021 | warning] aiogram: Updates were skipped successfully.
[Jan 08 17:23:07 2021 | info] bot: Starting demons...
[Jan 08 17:23:07 2021 | info] bot: Listening...
[Jan 08 17:23:24 2021 | info] aiogram.dispatcher.dispatcher: Start polling.
[Jan 08 17:25:04 2021 | info] bot: Shutdown... Please wait!
[Jan 08 17:25:04 2021 | info] aiogram.dispatcher.dispatcher: Stop polling...
[Jan 08 17:25:04 2021 | warning] aiogram: Goodbye!
[Jan 08 17:25:13 2021 | info] bot: Starting...
[Jan 08 17:25:15 2021 | info] aiogram: Bot: Cifra [@cifra_s_bot]
[Jan 08 17:25:16 2021 | warning] aiogram: Updates were skipped successfully.
[Jan 08 17:25:16 2021 | info] bot: Starting demons...
[Jan 08 17:25:16 2021 | info] bot: Listening...
[Jan 08 17:25:32 2021 | info] aiogram.dispatcher.dispatcher: Start polling.
[Jan 08 17:26:26 2021 | info] bot: Shutdown... Please wait!
[Jan 08 17:26:26 2021 | info] aiogram.dispatcher.dispatcher: Stop polling...
[Jan 08 17:26:26 2021 | warning] aiogram: Goodbye!
[Jan 08 17:26:36 2021 | info] bot: Starting...
[Jan 08 17:26:37 2021 | info] aiogram: Bot: Cifra [@cifra_s_bot]
[Jan 08 17:26:39 2021 | warning] aiogram: Updates were skipped successfully.
[Jan 08 17:26:39 2021 | info] bot: Starting demons...
[Jan 08 17:26:39 2021 | info] bot: Listening...
[Jan 08 17:27:00 2021 | info] aiogram.dispatcher.dispatcher: Start polling.
